vocabularies/security-classifications.ttl

```diff
@@ -0,0 +1,71 @@
+ @prefix qgiscf: <http://linked.data.gov.au/def/qg-security-classifications/> .
+
+ @prefix dc: <http://purl.org/dc/elements/1.1/> .
+ @prefix dct: <http://purl.org/dc/terms/> .
+ @prefix prov: <https://www.w3.org/TR/prov-o/> .
+ @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+ @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+ @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+ @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+ @prefix sdo: <http://www.schema.org/> .
+ @prefix owl: <http://www.w3.org/2002/07/owl#> .
+
+<http://linked.data.gov.au/def/qg-security-classifications> a owl:Ontology , skos:ConceptScheme ;
+dct:created "2020-11-11"^^xsd:date ;
+dct:creator <http://linked.data.gov.au/def/gsq-alias/des> ;
+dct:modified "2020-11-11"^^xsd:date ;
+dct:publisher <http://linked.data.gov.au/def/gsq-alias/des> ;
+dct:source "Qld Govt Information Security Policy (IS18:2018) and the Queensland Government Information Security Classification Framework (QGSCIF)"@en ;
+skos:definition "This is a vocabulary collection used by the DES data catalogue to determine the security classifications for the dataset.  It is utlised by the dataset: classifcation and access restrictions property."@en ;
+skos:prefLabel "QG Information Security Classifications"@en ;
+rdfs:label   "Qld Gov Information Security Classification"@en ;
+skos:hasTopConcept 
+    qgiscf:official ,
+    qgiscf:sensitive  ,
+    qgiscf:protected .
+
+<http://linked.data.gov.au/def/gsq-alias/des>
+    a sdo:Organization ;
+    sdo:name "Department of Environment and Science"@en .
+
+qgiscf:official a skos:Concept ; 
+ skos:topConceptOf <http://linked.data.gov.au/def/qg-security-classifications> ; 
+ skos:definition "OFFICIAL represents most Queensland Government information by volume, but lowest business impact per document if compromised or lost. However, where information is aggregated on an information asset such as an ICT server, the impact of compromise may increase and with it, the controls.  OFFICIAL information is routine information without special sensitivity or handling requirements. All routine public-sector business, operations and services is treated as OFFICIAL. At the OFFICIAL classification there is a general presumption that data may be shared across government. Security measures should be proportionate and driven by the business requirement.  Most OFFICIAL information is subject to the Public Records Act 2002 (Qld)."@en ;
+ skos:inScheme <http://linked.data.gov.au/def/qg-security-classifications> ;
+ skos:notation "off" ;
+ rdfs:label "OFFICIAL"@en ;
+ skos:prefLabel "OFFICIAL"@en  .
```

Comment:
Do we want all caps in our prefLabel?

Reply:
It's what the framework guideline suggests